setup.py

```diff
@@ -22,7 +22,7 @@
     long_description = fh.read()
 
 requirements = [
-    "qiskit>=0.32,<0.46",
+    "qiskit>=0.32",
```

Comment:
We no longer need an upper bound here? 🤔

Reply:
The issue was that we don't want to break people's environment if they install 1.0, and then install the plugin with the intention of using it for conversion. We want Qiskit 1.0 plus the PL-plugin for conversion to be an acceptable combination.

But this does mean if people pip install without an older version of Qiskit installed, then the devices won't work. Is there a more complicated version of a setup specification that allows us to say "install version 0.45.3 if they don't have Qiskit, but if they already have it and its above 0.32, leave their installation"? I.e. to have the default installed dependency differ from the highest allowed version?

Reply:
It's worth noting that `setup.py` only defines which versions of packages (e.g., Qiskit) are marked as "compatible" with the current package (e.g, the PL plugin). Consequently,
* If a user attempts to `pip install` the PL plugin and doesn't have Qiskit installed, `pip` will automatically search for a version of Qiskit which satisfies all of the constraints of the user's packages and install it.
* If a user attempts to `pip install` the PL plugin and already has a compatible version of Qiskit installed, nothing changes about their Qiskit installation.
* If a user attempts to `pip install` the PL plugin and has a version of Qiskit installed which is _not_ compatible, `pip` will attempt to reinstall a compatible version of Qiskit (as defined by the range in the PL plugin).

This is to say that we should only put an upper bound here if this plugin will not work with Qiskit 1.0 or later.

Reply:
After speaking to @trbromley offline, the decision was to remove the upper bound (we want people to be able to use Qiskit 1.0 and the conversion functionality), and update the install instructions so that if you just follow the instructions, you get `qiskit<0.46` and everything works. This means some people might have issues if they just pip install the plugin without looking at the documentation and they don't have a previous version of Qiskit and they want to use the devices. But something has to not work.